package.json

```diff
@@ -4,24 +4,29 @@
   "devDependencies": {
     "eslint": "^7.0.0",
     "eslint-config-google": "^0.14.0",
-    "prettier-eslint-cli": "^5.0.0",
-    "prettier-stylelint": "^0.4.2",
```

Comment:
this was pinning up on an old version of stylelint, which depends on an old version of braces, which was warning about a potential security issue.